console/src/app/pages/projects/apps/app-create/app-create.component.scss

```diff
@@ -50,6 +79,15 @@ p.desc {
   .step-description {
     font-size: 0.9rem;
   }
+
+  .mat-step-icon-content {
+    position: absolute;
+    transform: translate(-50%, -50%);
+    display: flex;
+    // fix for misplacement
+    top: 45%;
```

Comment:
<img width="129" alt="Bildschirm­foto 2023-04-06 um 14 34 30" src="https://user-images.githubusercontent.com/10165752/230380044-19ccd385-cb73-480d-9f4e-676935d701b2.png">
lets give it 50% and 50% so its aligned perfectly